ParisBNF/abb/BNFabb91.xml

```diff
@@ -290,10 +284,10 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                       <item xml:id="a2">
                          <locus from="86va" to="86vb"/>
                          <desc type="Unclear">A note written in an old crude hand mentioning ʾabuna Takla Ṣǝyon, probably a 
-                            senior local ecclesiastic. It refers to <foreign xml:lang="gez">መጽሐፈ፡ ተጋንዮ፡</foreign>, 
+                            local senior ecclesiastic. It refers to <foreign xml:lang="gez">መጽሐፈ፡ ተጋንዮ፡</foreign>, 
                             but the content is not fully clear and its relation to the manuscript is not obvious.</desc>
-                         <q xml:lang="gez">በመዋዕኢሁ፡ ለአቡነ፡ ተክለ፡ ጽዮን፡ ተጻፈ፡ ዝንቱ፡ በስመ፡ እግዚአብሔር፡ ሕቡረ፡ ሀልዎ፡ ልዑል፡ 
-                            ዘኢይረክብዎ፡ ክቡት፡ ዘኢይሬእይዎ፡ እሳት፡ ዘኢይለክፍዎ፡ መንፈስ፡ ዘኢገስስዎ፡ <gap reason="ellipsis" resp="DN"/></q> 
+                         <q xml:lang="gez">በመዋዕኢሁ፡ ለአቡነ፡ ተክለ፡ ጽዮን፡ <sic></sic>ዝንቱ፡ በስመ፡ እግዚአብሔር፡ ሕቡረ፡ ሀልዎ፡ ልዑል፡ 
```

Comment:
Sic has no content, so this mark up is not very clear

Reply:
Please merge it.